### PR TITLE
Feature/fab 47227

### PR DESCRIPTION
**Pull request by MKJSmith**

### CODE REVIEW

File: src/Controller/HealthCheckController.php [hunk 1]
```diff
@@ -72,37 +72,20 @@ protected function getBuildTimeForHumans(): ?string
...
         if ($this->doctrineOrm === null && $this->doctrineMongo === null) {
-            return ['N/A'];
+            return false;
         }
 
         if ($this->doctrineOrm !== null) {
-            try {
-                return [
-                    'type' => 'orm',
-                    'status' => $this->doctrineOrm->getConnection()->connect() ? 'connected' : 'disconnected'
```
Comment (MKJSmith) [inline src/Controller/HealthCheckController.php]: @felipeldn you still want some form of test of the connection, so here you could return: 

```return $this->doctrineOrm->getConnection()->connect()```

File: src/Controller/HealthCheckController.php [hunk 2]
```diff
@@ -72,37 +72,20 @@ protected function getBuildTimeForHumans(): ?string
...
-            }
+            return true;
         }
 
         if ($this->doctrineMongo !== null) {
-            try {
-                return [
-                    'type' => 'mongodb',
-                    'status' => 'connected',
-                    'databases' => $this->doctrineMongo->getConnection()->listDatabaseNames()
```
Comment (MKJSmith) [inline src/Controller/HealthCheckController.php]: And here, 

```
return is_array($this->doctrineMongo->getConnection()->listDatabaseNames();
```

**Review by MKJSmith — COMMENTED** [review]
a small tweak needed in the db check logic

File: src/Controller/HealthCheckController.php (hunk 2 above)
Comment (felipeldn) [reply src/Controller/HealthCheckController.php]: Still having a play around, but pushed these changes 👍

**Pull request merged**